spot-contracts/contracts/FeePolicy.sol

```diff
@@ -149,33 +153,41 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     /// @notice Updates the target subscription ratio.
     /// @param targetSubscriptionRatio_ The new target subscription ratio as a fixed point number with {DECIMALS} places.
     function updateTargetSubscriptionRatio(uint256 targetSubscriptionRatio_) external onlyOwner {
-        require(targetSubscriptionRatio_ > SR_LOWER_BOUND, "FeeStrategy: sr too low");
-        require(targetSubscriptionRatio_ <= SR_UPPER_BOUND, "FeeStrategy: sr high low");
+        if (targetSubscriptionRatio_ < SR_LOWER_BOUND || targetSubscriptionRatio_ > SR_UPPER_BOUND) {
+            revert InvalidTargetSRBounds();
+        }
         targetSubscriptionRatio = targetSubscriptionRatio_;
     }
 
     /// @notice Updates the perp mint fee parameters.
     /// @param perpMintFeePerc_ The new perp mint fee ceiling percentage
     ///        as a fixed point number with {DECIMALS} places.
     function updatePerpMintFees(uint256 perpMintFeePerc_) external onlyOwner {
-        require(perpMintFeePerc_ <= ONE, "FeeStrategy: perc too high");
+        if (perpMintFeePerc_ > ONE) {
+            revert InvalidPerc();
+        }
         perpMintFeePerc = perpMintFeePerc_;
     }
 
     /// @notice Updates the perp burn fee parameters.
     /// @param perpBurnFeePerc_ The new perp burn fee ceiling percentage
     ///        as a fixed point number with {DECIMALS} places.
     function updatePerpBurnFees(uint256 perpBurnFeePerc_) external onlyOwner {
-        require(perpBurnFeePerc_ <= ONE, "FeeStrategy: perc too high");
+        if (perpBurnFeePerc_ > ONE) {
+            revert InvalidPerc();
+        }
         perpBurnFeePerc = perpBurnFeePerc_;
     }
 
     /// @notice Update the parameters determining the slope and asymptotes of the sigmoid fee curve.
     /// @param p Lower, Upper and Growth sigmoid paramters are fixed point numbers with {DECIMALS} places.
     function updatePerpRolloverFees(RolloverFeeSigmoidParams calldata p) external onlyOwner {
-        require(p.lower >= -int256(SIGMOID_BOUND), "FeeStrategy: sigmoid lower bound too low");
-        require(p.upper <= int256(SIGMOID_BOUND), "FeeStrategy: sigmoid upper bound too high");
-        require(p.lower <= p.upper, "FeeStrategy: sigmoid asymptotes invalid");
+        // SIGMOID_BOUND is set to 1%, i.e) the rollover fee can be at most 1% on either direction.
+        // If the bond duration is 28 days and 13 rollovers happen per year,
```

Comment:
I think any comments related to numbers should be where those numbers are defined. So where SIGMOID_BOUND is defined.

In the logic where they're used, we treat them as givens.

Reply:
Right moved